core/src/main/scala/org/bitcoins/core/protocol/dlc/CETCalculator.scala

```diff
@@ -51,18 +51,20 @@ object CETCalculator {
     * (Note that interpolated functions are allowed
     * to be negative, but we set all negative values to 0).
     */
-  case class StartZero(indexFrom: Long, indexTo: Long) extends CETRange
+  case class RemotePayoutRange(indexFrom: Long, indexTo: Long) extends CETRange
```

Comment:
Local/Remote language should be reserved for values which differ between the two parties where there is a clear "local" actor. `RemotePayoutRange` is very misleading because all payouts from this file are from the local point of view. I'm not attached to `StartZero` and `StartFuncConst` but I find the use of `Remote` and `Local` in these two new names much more confusing than those.